articleController.js

```diff
@@ -1,5 +1,19 @@
 var articleController = {};
 
 articleController.index = function () {
-  blog.determineData(); 
+  blog.determineData();
+};
+
+articleController.category = function (ctx, next) {
+  var categoryData = function (data) { //rename data to articles?  it is the return of .findByCategory
+    ctx.articles = data;
+    next();
+  };
+  CompleteArticle.findByCategory(ctx.params.category, categoryData);
```

Comment:
Oh, you have `ctx.params.category` here. Hmm well as we can see above though, categoryData is broken, so this won't work.